onnxruntime/test/providers/cpu/tensor/grid_sample_test.cc

```diff
@@ -0,0 +1,1019 @@
+#include "gtest/gtest.h"
+#include "test/providers/provider_test_utils.h"
+#include "test/util/include/default_providers.h"
+
+#include <limits>
```

Comment:
It would be great to apply clang-format -i <filename> to avoid long lines.

Reply:
Understood. Allow me to do this in my next PR because rerun the CI may take over a day given possible CI unstable failures.

Reply:
@xadupre  I ran lintrunner on grid_sample_test_gen.py and did not see issue with the python code. gird_sample_test.cc is generated code.